src/containerapp/azext_containerapp/commands.py

```diff
@@ -13,7 +13,7 @@
                             transform_job_execution_list_output,
                             transform_job_execution_show_output,
                             transform_revision_list_output,
-                            transform_revision_output)
+                            transform_revision_output, transform_usages_output)
```

Comment:
```suggestion
                            transform_revision_output, 
                            transform_usages_output)
```

Reply:
current code also aligns with pylint check. I think it is fine.